ipld/write.go

```diff
@@ -13,13 +13,20 @@ import (
 	"github.com/celestiaorg/celestia-core/pkg/wrapper"
 )
 
+// BatchSize defines an amount of IPLD Nodes to be buffered and written at once.
+// This configuration is very database backend specific and the current default(128) is not optimized for the version of
+// Badger we use. We set it to one to avoid test flakiness, as some tests may read for data that was not written yet.
+// TODO(@Wondertan): Find out the perfect value for Badger(e.g. ask PL folks) or migrate to go-car IPLD
+//  storage(preferred).
+const BatchSize = 1
```

Comment:
Is there a reason to set this to `1` when the comment says `128`?

Reply:
128 is set by default, so when we don't set anything it will be 128, but purposely change it to 1 and the purpose is described.